api/types/mount/mount.go

```diff
@@ -109,7 +109,8 @@ type TmpfsOptions struct {
 	SizeBytes int64 `json:",omitempty"`
 	// Mode of the tmpfs upon creation
 	Mode os.FileMode `json:",omitempty"`
-
+	// Options passed directly to the tmpfs mount
+	Options string `json:",omitempty"`
```

Comment:
Can you add this to the Swagger.yml as well, and to the API history https://github.com/moby/moby/blob/b4842cfe88b39af42416429396aee10d1fe7c3d6/docs/api/version-history.md#v140-api-changes? https://github.com/moby/moby/blob/b4842cfe88b39af42416429396aee10d1fe7c3d6/api/swagger.yaml#L297-L307


@kolyshkin did we want this to be a `[]string` or `map[string]string` ?

Reply:
Thank you for the review. I updated the `swagger.yaml` and the `version-history.md` files, assuming `string` for the `Options` field as I recall from previous discussions. If a `map[string]string` is to be used instead, please let me know.